### DISCUSSION TITLE
My request keeps getting denied

### DISCUSSION THREAD

**Discussion by rakkdur** [community/community#20691] [GitHub Education]
Hey there,

I am a student at Western Governors University and I hav applied 3 times, it won’t accept my unofficial transcripts, my class schedule, my classes or anything. What am I doing wrong and why do I keep getting flagged as all my documents aren’t allowed any help with this matter would be amazing

**Comment by melvincwng** [comment]
@rakkdur , try raising a ticket to GitHub Education Support over [here](https://support.github.com/contact/education) for them to check your issue.

Hope this clarifies.

**Reply by rakkdur** [reply]
@melvincwng I actually figured it out I actually needed to do a dual authentication at one time, I had to do my student ID AND my unofficial transcript at the same time in the same picture

thank you very much for your response 4th times the charm

**Reply by melvincwng** [reply]
Awesome! Glad this issue has been resolved for you :)
